aiReachability/aiReachability/NetworkMonitor.swift

```diff
@@ -37,7 +44,12 @@ public class NetworkMonitor {
     }
 
     private func handleUpdate(path: NWPath) {
-        wifiState = path.status == .satisfied ? .connected : .disconnected
-        cellularState = path.isExpensive ? .connected : .disconnected
+        let reachabilityState = ReachabilityState(path: path)
+        wifiState = reachabilityState.contains(.wifi) ? .connected : .disconnected
+        cellularState = reachabilityState.contains(.cellular) ? .connected : .disconnected
+
+        DispatchQueue.main.async {
```

Comment:
Does it really to be done in the main thread?

Reply:
SDK should deliver on main thread